scalableminds/webknossos
# Register all segments in a bounding box

### URL of deployed dev instance (used for testing):
- https://registerallsegmentsinbb.webknossos.xyz/

### Steps to test:
- Go to bounding box tab 
- inspect new overflow menu
- make sure everything works as before
- try out new action: "register all segments for bounding box"
- test segment and data limits

## TODOs
- [x] add context menu
- [x] add new entry with functionality
- [x] include limits on segments and data
- [x] add constant
- [x] add padding to BB entries
- [x] fix spacing
- [x] add registered segments to group
- [x] test that all the old disabling of context menu entries is still working
- [x] fix segment 0 bug 
- [x] disable if allowupdate is false

### Issues:
- fixes #7906 

------
(Please delete unneeded items, merge only when none are left open)
